app/models/vm_or_template/right_sizing.rb

```diff
@@ -163,16 +163,8 @@ def max_mem_usage_absolute_average_max_over_time_period
   end
 
   def cpu_usagemhz_rate_average_max_over_time_period
-    if has_attribute?("cpu_usagemhz_rate_average_max_over_time_period")
-      self["cpu_usagemhz_rate_average_max_over_time_period"]
-    else
-      opts = {
-        :end_date => Time.now.utc,
-        :days     => Metric::LongTermAverages::AVG_DAYS
-      }
-      VimPerformanceAnalysis.find_perf_for_time_period(self, "daily", opts)
-                            .maximum(:cpu_usagemhz_rate_average)
-    end
+    perfs = VimPerformanceAnalysis.find_perf_for_time_period(self, "daily", :end_date => Time.now.utc, :days => Metric::LongTermAverages::AVG_DAYS)
+    perfs.collect(&:abs_max_cpu_usagemhz_rate_average_value).compact.max
```

Comment:
so we are taking max ```abs_max_cpu_usagemhz_rate_average_value``` instead of ```cpu_usagemhz_rate_average```

Reply:
`cpu_usagemhz_rate_average` is the average `mhz usage` for that rollup period.
`abs_max_cpu_usagemhz_rate_average_value` is added in this PR and is capturing the `abs max` ever recorded in this period. It is determined [here](https://github.com/jameswnl/manageiq/blob/6c745bb3b28ede2552a9446ed6f4074c4698326b/app/models/metric/rollup.rb#L243-L243)
(See the bug report's observation)